agent/src/beerocks/monitor/monitor_thread.cpp

```diff
@@ -53,6 +53,8 @@ monitor_thread::monitor_thread(const std::string &slave_uds_, const std::string
 
     using namespace std::placeholders; // for `_1`
 
+    network_utils::linux_iface_get_mac(bridge_iface, al_mac);
```

Comment:
You can mention in the commit message why you need the al_mac in the monitor and that this is a preparative commit.

Reply:
I go over the other commits, and I didn't notice it serves any purpose changing it here and on the other follow-up commits. Also, it doesn't right to call it al_mac here, it is needed for arp_monitor where the bridge interface required.
al_mac does not logically have a single bridge mac.
So all of the commits related to changing bridge to al_mac should be dropped.

Reply:
Indeed, the "bridge_4addr_mac" is a brittle stack of undocumented hacks that shouldn't be changed...

Bottom line is that we still have to set it with a VS message, together with the ipv4 address. They are used together.